src/components/skills/SkillIcon.tsx

```diff
@@ -83,7 +87,23 @@ function SkillIcon({
 
   return (
     <SkillIconTooltipTemplate tooltip={t(identifier)}>
-      <Icon className={`h-4 w-4 ${className}`} />
+      <span
+        className={cx(
+          'rounded-full opacity-75',
+          size === 'small' && 'max-h-4 min-h-4 min-w-4 max-w-4 p-0.5',
+          size === 'medium' && 'max-h-5 min-h-5 min-w-5 max-w-5 p-[3px]',
+          size === 'large' && 'max-h-6 min-h-6 min-w-6 max-w-6 p-1',
+          className,
+        )}
+      >
+        <Icon
+          className={cx(
+            size === 'small' && 'h-3 w-3',
+            size === 'medium' && 'h-3.5 w-3.5',
+            size === 'large' && 'h-4 w-4',
+          )}
```

Comment:
A little easier to read:
```suggestion
          className={cx({
            'h-3 w-3': size === 'small',
            'h-3.5 w-3.5': size === 'medium',
            'h-4 w-4': size === 'large',
          })}
```